temel_bilgiler/html_css/kutu-modeli/kutu-modeli.md

```diff
@@ -25,45 +25,45 @@ Every single thing on a webpage is a rectangular box. These boxes can have other
 
-The only real complication here is that there are many ways to manipulate the size of these boxes, and the space between them, using `padding`, `margin`, and `border`. The assigned articles go into more depth on this concept, but to sum it up briefly:
+Buradaki tek gerçek karmaşıklık, `iç kenar boşluğu`, `dış kenar boşluğu` ve `kenar çizgisi` kullanarak bu kutuların boyutlarını ve aralarındaki boşlukları değiştirmenin birçok yolunun olmasıdır. Bu kavrama özel makaleler, kavramı daha derinlemesine ele alıyor, ancak kısaca özetlememiz gerekirse:
 
-- `padding` increases the space between the border of a box and the content of the box.
-- `margin` increases the space between the borders of a box and the borders of adjacent boxes.
-- `border` adds space (even if it's only a pixel or two) between the margin and the padding.
+- `iç kenar boşluğu`, bir kutunun kenarları ile kutunun içeriği arasındaki boşluğu artırır.
+- `dış kenar boşluğu`, bir kutunun kenarları ile bitişiğindeki kutuların kenarları arasındaki boşluğu artırır.
+- `kenar çizgisi`, dış kenar boşluğu ile iç kenar boşluğu arasına bir boşluk ekler (yalnızca birkaç piksel olsa bile).
 
-Be sure to study the diagrams carefully.
+Diyagramları dikkatlice incelediğinizden emin olun.
 
 ![the box model](https://cdn.statically.io/gh/TheOdinProject/curriculum/main/foundations/html_css/the-box-model/imgs/box-model.png)
 
-### Assignment
+### Ödev
 
 <div class="lesson-content__panel" markdown="1">
 
-1. [This video](https://www.youtube.com/watch?v=rIO5326FgPE) is a straightforward overview of the box model, padding and margin. Go ahead and watch this now; it informs everything else.
-2. Because the box model concept is so incredibly fundamental, check out [this lesson from MDN](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/The_box_model). It covers the same material as the video above, but it goes a little further in-depth. Pay close attention to the examples and take the time to experiment with their in-browser editor!
-3. [This CSS Tricks page](https://css-tricks.com/almanac/properties/m/margin/) has some further information about the `margin` property that you'll find useful. Specifically, the sections about `auto` and margin collapsing contain things you'll want to know.
+1. [Bu video](https://www.youtube.com/watch?v=rIO5326FgPE), kutu modeli, iç kenar boşluğu ve dış kenar boşluğu konularına basit bir genel bakış niteliğindedir. Haydi şimdi bu videoyu izleyin; sizi bu konunun diğer tüm yönleri hakkında bilgilendirecektir.
+2. Kutu modeli kavramı son derece temel bir kavram olduğu için, [MDN'deki bu derse](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/The_box_model) göz atın. Yukarıdaki video ile aynı materyali kapsar, ancak konunun biraz daha derinine iner. Verilen örneklere çok dikkat edin ve tarayıcı içi düzenleyicilerini denemeye vakit ayırın!
```

Comment:
fix: Kutu modeli kavramı son derece temel bir kavram olduğu için, [MDN'deki bu İngilizce dilindeki derse](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/The_box_model) göz atın.